apps/web-giddh/src/app/gst/gstR3/gstR3.component.scss

```diff
@@ -408,9 +408,9 @@ fieldset[disabled] .btn-primary:hover .btn-primary[disabled],
 }
 
 .btn-primary {
-    color: #fff;
-    background-color: #d64830;
-    border-color: #c53e27;
+    color: $white;
+    background-color: $cinnabar-orange;
```

Comment:
In this html file, the button on which this class is applied is commented, but we can't remove this color from color.scss because this is used in other file